core-feature-pack/src/main/resources/content/bin/standalone.conf.ps1

```diff
@@ -62,7 +59,7 @@ $JAVA_OPTS += '-Dorg.jboss.resolver.warning=true'
 
 # Make Byteman classes visible in all module loaders
 # This is necessary to inject Byteman rules into AS7 deployments
-$JAVA_OPTS += '-Djboss.modules.system.pkgs=org.jboss.byteman'
+$JAVA_OPTS += "-Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS"
```

Comment:
I've reworked how all this works, so it is not as confusing as it was, and works much better when tested.